.github/pull_request_template.md

```diff
@@ -18,3 +18,10 @@ Please tick as appropriate and edit the reasons (e.g.: "backport: not needed bec
 
 Please refer to the [contributing guidelines](https://github.com/Mbed-TLS/mbedtls/blob/development/CONTRIBUTING.md), especially the
 checklist for PR contributors.
+
+Help make review efficient:
+* Multiple simple commits
+  - please structure your PR into a series of small commits, each of which does one thing
+* No force-push
```

Comment:
I wouldn't go that far. I'd say the _normal_ way of updating if by adding new commits, and force-pushing is the exception and should be agreed with reviewers. (In particular, I think in our usage, for resolving conflicts, rebasing is more common than merging.)

Reply:
I've very slightly softened this suggestion. But in general, I think we have difficulties from unwanted force-pushing far more than the opposite. Plus, it's easy as a reviewer to say "please force-push to tidy up these commits" - but going the other direction doesn't really work.

So, in the interest of keeping this message simple enough that casual contributors are likely to follow it, I think we should just advise against force-pushing.